.github/workflows/cmake.yml

```diff
@@ -284,7 +284,7 @@ jobs:
           -DLIBINT2_PYTHON=ON \
```

Comment:
Lori suggested turning this OFF if we're _still_ having problems even after this latest version.